stratum/hal/lib/tdi/es2k/es2k_hal.cc

```diff
@@ -142,6 +143,16 @@ ::util::Status Es2kHal::Setup(bool warmboot) {
 
   RETURN_IF_ERROR(RecursivelyCreateDir(FLAGS_persistent_config_dir));
 
+  // Create a new, empty pipeline configuration file. On startup, the P4 service checks
+  // for an existing pipeline configuration and reapplies it. For ES2K, we want to come
+  // up with no configuration and wait for the client to supply one.
+  FILE* pipeline_cfg_file =
+    fopen(FLAGS_forwarding_pipeline_configs_file.c_str(), "wb");
+  if (pipeline_cfg_file != NULL) {
+    LOG(INFO) << "Truncating saved pipeline configuration file.";
```

Comment:
Do we need the info message in this case? Is this something we should do silently?

Reply:
I think we should log a message and not do this silently since the behavior varies between Tofino and DPDK/ES2K target.
But messaging should be clear and indicate what we are doing.

Thoughts?